src/test/java/com/quirko/gui/GuiControllerTest.java

```diff
@@ -0,0 +1,64 @@
+package com.quirko.gui;
+
+import javafx.util.Pair;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+import static org.junit.Assert.*;
+
+public class GuiControllerTest {
+
+    @Test
+    public void testScoreboardSize() {
+        Scanner file = null;
+        try {
+            file = new Scanner(new File("score.txt"));
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
+        int numLines = 0;
+        while (file.hasNextLine()) {
+            numLines++;
+            file.nextLine();
+        }
+
+        GuiController guiController = new GuiController();
+        guiController.loadScores();
+        assertEquals(guiController.getScores().size(), numLines);
+    }
+
+    @Test
+    public void testNullScore() {
+        GuiController guiController = new GuiController();
+        guiController.loadScores();
+        boolean noNull = true;
+
+        if (guiController.getScores() != null) {
```

Comment:
Burada neden if var?

Doğrudan for döngüsüne girsin. Eğer test if içerisine girmiyorsa herhangi bir şeyi test etmiyor.